.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Set up PureScript toolchain
         uses: purescript-contrib/setup-purescript@main
         with:
-          purescript: "0.14.0-rc5"
+          purescript: "0.14.0"
```

Comment:
As with your other PR, these two lines can be removed as setup-purescript will pull in the latest compiler automatically.